pkg/model/openstackmodel/servergroup.go

```diff
@@ -73,6 +73,15 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 		igUserData = fi.String(startupStr)
 	}
 
+	uniqueZones := make(map[string]string)
+	for _, subnet := range b.Cluster.Spec.Subnets {
+		uniqueZones[subnet.Zone] = ""
+	}
+	var zones []string
+	for k, _ := range uniqueZones {
+		zones = append(zones, k)
+	}
```

Comment:
is there simpler way to do this? So I want unique zone values from subnet zones

Reply:
Have a peek at the maps & slice util packages in the repo, you could reuse or extend the functionality there to make this a little cleaner.

Reply:
@KashifSaadat in my opinion its now cleaner :)